components/well-bold/well-bold-doc-en.html

```diff
@@ -0,0 +1,79 @@
+---
+breadcrumb: [
+	{ "title": "GCWeb home", "link": "https://wet-boew.github.io/GCWeb/home.html" }
+]
+dateModified: "2022-04-28"
+description: "Documentation and working example for the well bold"
+language: "en"
+altLangPage: "well-bold-doc-fr.html"
+title: "Well bold"
+---
+
+<span class="wb-prettify all-pre hide"></span>
+
+<dl class="horizontal">
+	<dt>Status</dt>
+	<dd>Stable</dd>
+	<dt>Type</dt>
+	<dd>Component</dd>
+	<dt>Last review</dt>
+	<dd>2022-04-28</dd>
+	<dt>Conforming to</dt>
+	<dd>n/a</dd>
+	<dt>Nom technique</dt>
+	<dd><code>well-bold</code></dd>
+</dl>
+
+<h2>Purpose</h2>
+<p>This apply font weight bold to evey text element inside a well component and generates normal font weight when the element <code>strong</code> is applied.</p>
+
+<h2>Working example</h2>
+<ul>
+	<li><a href="well-bold-en.html">Working example</a></li>
+</ul>
+
+<h3>Code</h3>
+<pre><code>
+&lt;section class="well well-bold"&gt;
```

Comment:
```suggestion
<pre><code>&lt;section class="well well-bold"&gt;
```

It will remove the initial blank line in the code sample

Reply:
Changed. Thanks